src/components/side_panel/table/table_panel.xml

```diff
@@ -0,0 +1,83 @@
+<templates>
+  <t t-name="o-spreadsheet-TablePanel">
+    <div class="o-table-panel">
+      <Section>
+        <t t-set-slot="title">Table options</t>
+        <div class="d-flex flex-row">
+          <div class="w-50">
+            <Checkbox
+              label="_t('Filter Button')"
```

Comment:
this unfortunately doesn't work because the source term will never be extracted by the extraction script. It's not looking at the (string) value of xml attributes, interpret it as javascript and look for `_t`. The script is quite naive and is a simple static extraction.

Options are:
- move that to the typescript `this.filterButtonLabel = _t('Filter Button')`
- define a variable in the template `<t t-set="filter_button_label">Filter Button</t>`. Here because `Filter Button` looks like regular text content, it is extracted, just like if it was `<span>Hello world</span>`